tests/instantiation/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
+1 for `instantiation` tests package -  even though we dont have an `instanitation` code package yet, this is where I too think the tests should reside